Add optional "geometry_type" field per column (#51)
opengeospatial/geoparquet: format-specs/geoparquet.md

| ---------- | ----------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------- |
| crs       | string   | **REQUIRED** [WKT2](https://docs.opengeospatial.org/is/18-010r7/18-010r7.html) string representing the Coordinate Reference System (CRS) of the geometry.  |
| encoding | string | **REQUIRED** Name of the geometry encoding format. Currently only 'WKB' is supported. |

> | geometry_type | string or \[string] | **OPTIONAL** If known, the geometry type(s) of all geometries.  |

Reviewer: ```suggestion
| geometry_type | string or \[string] | **REQUIRED** The geometry type(s) of all geometries, or "Unknown" if they are not known.  |
```

(x, y) where x is easting or longitude and y is northing or latitude. This ordering explicitly overrides the axis order as specified in the CRS. 
This follows the precedent of [GeoPackage](https://geopackage.org), see the [note in their spec](https://www.geopackage.org/spec130/#gpb_spec). 

#### geometry_type

> This optional field captures the geometry type(s) of the geometries in the

Reviewer: ```suggestion
This field captures the geometry type(s) of the geometries in the
```

column, when known. Accepted geometry types are: "Point", "LineString",
"Polygon", "MultiPoint", "MultiLineString", "MultiPolygon",
"GeometryCollection".

In addition, the following rules are used:

- In case of 3D geometries, a " Z" suffix gets added (e.g. "Point Z").
- The value can be a single string or an array of strings in case multiple
  geometry types are present (e.g. ["Polygon", "MultiPolygon"]).
- Additionally the value "Unknown" is accepted to explicitly signal that the
  geometry type is not known.

> If specified, it is expected that this field is strictly correct. For

Reviewer: ```suggestion
It is expected that this field is strictly correct. For
```
